api/config/token.ts

```diff
@@ -1,14 +1,24 @@
-const jwt = require("jsonwebtoken");
+import jwt, { JwtPayload } from "jsonwebtoken";
 
-const SECRET = process.env.SECRET_PASSWORD;
+const SECRET = process.env.SECRET_PASSWORD || "";
```

Comment:
No. No. No. Can we work without SECRET? No, then app should break, so no defaulting here.